Documentation/MigrateToComposer/MigrationSteps.rst

```diff
@@ -329,6 +329,8 @@ To complete our example :file:`composer.json`, it would look like this:
             ]
         }
     }
+    
+After adding files to the autoload you should run `composer dumpautoload`. This command will re-generate the vendor/autoload.php file
```

Comment:
This should be `paths` instead of files since we suggest PSR-4. Also I wouldn't directly mention a specific autoload file but simply talk about the "autoload info" or "autoload registry".

Reply:
So like "After adding paths to the autoload you should run `composer dumpautoload`. This command will re-generate the autoload info and should be run anytime you add new paths to the autoload portion in the composer.json"

Reply:
Sounds good :+1: